### PR TITLE
fix: use template id to list pipeline template versions

### PR DESCRIPTION
**Pull request by klu909**
## Context

attempting to list pipeline template version would throw `Error: Invalid param "namespace"`

## Objective
we only need template id to list pipeline template, name and namespace are not necessary 

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Review by VonnyJap — APPROVED** [review]

File: lib/pipelineTemplateVersionFactory.js [hunk 1]
```diff
@@ -162,9 +162,11 @@ class PipelineTemplateVersionFactory extends BaseFactory {
...
 
-        config.params.templateId = pipelineTemplateMeta.id;
-
-        return super.list(config);
+        return super.list({
+            params: {
+                templateId: pipelineTemplateMeta.id
+            }
+        });
     }
```
Comment (pritamstyz4ever) [inline lib/pipelineTemplateVersionFactory.js:170]: revert this change, we need to pass config as it can contain, sort and paginate

**Review by pritamstyz4ever — COMMENTED** [review]
Revert this change
